packages/teleport/src/services/audit/types.ts

```diff
@@ -1139,6 +1157,15 @@ type RawEventData<T extends EventCode> = RawEvent<
   }
 >;
 
+type RawDeviceTrust<T extends EventCode> = RawEvent<
```

Comment:
We may want to call this RawDeviceEvent, assuming we keep the current names in the other PR.